oe_content.install

```diff
@@ -37,3 +39,17 @@ function oe_content_update_8001() {
     $entity_definition_update_manager->updateFieldStorageDefinition($field_definition);
   }
 }
+
+/**
+ * Update field storage with oe_reference.
+ */
+function oe_content_update_8002() {
```

Comment:
We stopped using update hooks as you might not have access to the full Drupal API here. Move this in `oe_content.post_update.php` and rename it following the convention there.

Reply:
Fixed